### ISSUE TITLE
Invalid highlighting for NBT array tags

### ISSUE THREAD

**Issue by SPGoding** [Arcensoth/language-mcfunction#30]
![image](https://user-images.githubusercontent.com/15277496/69885384-767b1080-12a2-11ea-8c55-3582b469a6af.png)

```mcfunction
give @s minecraft:stone{foo: [B; 123b, 123b]}
give @s minecraft:stone{foo: [I; 123, 456]}
give @s minecraft:stone{foo: [L; 123L, 456L]}
```

**Comment by Arcensoth** [comment]
Now that you mention it, I don't think I ever even considered typed arrays. Thanks for bringing this to my attention. Will be fixed in the next release.

**Comment by Arcensoth** [comment]
Fixed in 86d8e8b
